ci.sh

```diff
@@ -110,12 +110,10 @@ if rustup component list | grep -q clippy; then
     cargo clippy --all-targets $FEATURES -- -D clippy::all -D clippy::cargo
 fi
 
-echo "Unit tests"
-RUST_BACKTRACE=1 cargo test --lib $FEATURES
-echo "Doc tests"
-RUST_BACKTRACE=1 cargo test --doc $FEATURES
+echo "Unit, doc and integration tests"
+RUST_BACKTRACE=1 cargo test $FEATURES
```

Comment:
I am wondering if there will be any problem with the `mappings` folders created in the integration tests, if it ever clashes with the ones in the end-to-end tests. Maybe it would be nice to rename them `PROVIDER_mappings` and/or delete them at the end?

Reply:
makes sense, will remove the dir after these tests!